src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -141,6 +141,10 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
         [isSmallScreenWidth, styles, renderItem, translate],
     );
 
+    useEffect(() => {
```

Comment:
We should also trigger this callback onReconnect, like we do [here](https://github.com/Expensify/App/blob/686705e43a95c897d50721f4e77652fbf008f99e/src/pages/workspace/categories/WorkspaceCategoriesPage.tsx#L63) for example

Reply:
done!